src/mmw/js/src/account/views.js

```diff
@@ -13,7 +15,63 @@ var Clipboard = require('clipboard'),
     accountTmpl = require('./templates/account.html');
 
 var ProfileView = Marionette.ItemView.extend({
-    template: profileTmpl
+    template: profileTmpl,
+
+    ui: {
+        firstName: '#first_name',
+        lastName: '#last_name',
+        organization: '#organization',
+        userType: '#user_type',
+        country: '#country',
+        postalCode: '#postal_code',
+        saveChanges: '[data-action="save_changes"]'
+    },
+
+    events: {
+        'click @ui.saveChanges': 'saveChanges'
+    },
+
+    initialize: function(){
+        this.listenTo(this.model, 'change', this.render);
+    },
+
+    onRender: function() {
+        var self = this;
+        function addOptions(model, field, selectedValue) {
+            $.each(window.clientSettings.choices[model][field], function(index, choice) {
+                var value = choice[0];
+                var text = choice[1];
+                var selected = value === selectedValue ? ' selected ' : '';
+                self.$el.find('#' + field).append('<option ' + selected + ' value="' + value + '">' + text + '</option>');
+            });
+            self.$el.find('#' + field).selectpicker();
+        }
+        addOptions('UserProfile', 'country', self.model.attributes.country);
+        addOptions('UserProfile', 'user_type', self.model.attributes.user_type);
+    },
+
+    saveChanges: function () {
+        var self = this;
+        self.model.save({
+            first_name: this.ui.firstName.val(),
+            last_name: this.ui.lastName.val(),
+            organization: this.ui.organization.val(),
+            user_type: this.ui.userType.val(),
+            country: this.ui.country.val(),
+            postal_code: this.ui.postalCode.val(),
+            error: null,
+            saving: true
+        },
+        {
+            success: function() {
```

Comment:
For stylistic consistency, prefer defining this using `done`, `fail`, and `always`:

```js
var model = this.model;

model.save({ ... })
     .done(function() {
         App.user.set('profile_is_complete', true);
     })
     .fail(function() {
         model.set('error', 'There was a problem saving your profile.');
     })
     .always(function() {
         model.set('saving', false);
     });
```

Reply:
Addressed in fixup 2259d26